test/blackbox-tests/test-cases/install/disable--p.t

```diff
@@ -0,0 +1,29 @@
+  $ cat > dune-project <<EOF
+  > (lang dune 3.2)
+  > (package
+  >   (name repro))
+  > (package
+  >    (name repro-other))
+  > EOF
+
+  $ cat > dune <<EOF
+  > (library
+  >  (name repro)
+  >  (public_name repro)
+  >  (modules foo))
+  > (library
+  >  (name repro_other)
+  >  (public_name repro-other)
+  >  (modules bar))
+  > EOF
+
+  $ cat > foo.ml <<EOF
+  > let x = 1
+  > EOF
+  $ cp foo.ml bar.ml
```

Comment:
Using `cp` is problematic sometimes on macos. Creating the file twice should be fine.